website/versions.json

```diff
@@ -0,0 +1,3 @@
+[
+  "0.4.0"
```

Comment:
I would like to avoid theses kind of static files, it's gonna be so painfull to maintain and not to forget to update them when a new version will be released.

Is it not possible to generate a list from a `git tag | grep -E ^v` command ?

![image](https://user-images.githubusercontent.com/6154987/107647289-49363a80-6c7b-11eb-8492-4f881e360bb4.png)

Reply:
It was generated by docusaurus as per their doc https://v2.docusaurus.io/docs/versioning#tagging-a-new-version